docs/support/faq.md

```diff
@@ -158,75 +158,75 @@ For other permissions, please see the contract [here](https://github.com/LIT-Pro
 
-Firstly, if you are using Lit in a browser, the AuthSig is stored in its local storage so you don’t need to sign multiple times as the stored AuthSig will be used subsequently. This design pattern is specifically applicable to when you want to sign different SIWE resources or messages which is generally required in [Custom Contract Calls](../sdk/access-control/evm/custom-contract-calls.md).
+Firstly, if you are using Lit in a browser, the AuthSig is stored in its local storage so you don't need to sign multiple times as the stored AuthSig will be used subsequently. This design pattern is specifically applicable when you want to sign different SIWE resources or messages, which is generally required in [Custom Contract Calls](../sdk/access-control/evm/custom-contract-calls.md).
 
-For different resources you’ll be required to sign each time since the signed message is different. One way to get around this is to use a PKP to do the signing for you. This is how it works: The user owns a PKP & uses it to sign a message in a LitAction with `Lit.Actions.signEcdsa()`. Then return all the sigShares from the LitAction to the app. Here the user can loop through & craft AuthSigs for each returned sigShare and process the required access control operation like decryption. Another way is to return the crafted AuthSigs from the LitActions directly.
+For different resources, you'll be required to sign each time since the signed message is different. One way to get around this is to use a PKP to do the signing for you. This is how it works: The user owns a PKP & uses it to sign a message in a LitAction with `Lit.Actions.signEcdsa()`. Then return all the sigShares from the LitAction to the app. Here the user can loop through & craft AuthSigs for each returned sigShare and process the required access control operation like decryption. Another way is to return the crafted AuthSigs from the LitActions directly.
 
 ### 5. Out of the above two approaches, which one is preferred?
 
-A key point to note here is that the LitAction can execute only for 60 seconds after which it times out. Hence it makes sense to reduce complexity in the LitActions code. Also if you want to create a lot of AuthSigs process those in batches. This will require multiple calls to execute the LitAction. You may also parallelize these promises using a package like [p-queue](https://www.npmjs.com/package/p-queue).
+A key point to note here is that the LitAction can execute only for 60 seconds after which it times out. Hence it makes sense to reduce complexity in the LitActions code. Also, if you want to create a lot of AuthSigs, process those in batches. This will require multiple calls to execute the LitAction. You may also parallelize these promises using a package like [p-queue](https://www.npmjs.com/package/p-queue).
 
 <br />
 
 ## Security & Trust Implications
 
 ### 1. What encryption algorithm are you using? AES?
```

Comment:
this needs to be updated for identity based encryption

Reply:
Changed as mentioned in the google doc